core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -66,6 +66,7 @@
     protected static final String FALSE_VALUE = "false";
     private static final String MAX_LIST_LENGTH = "100";
     private static final String MULIPLE_ATTRIBUTE_ENABLE = "MultipleAttributeEnable";
+    private static final String DISAPLAY_NAME_CLAIM = "http://wso2.org/claims/displayName";
```

Comment:
use the DEFAULT_CARBON_DIALECT for deriving the claim

private static final String DISPLAY_NAME_CLAIM = UserCoreConstants.DEFAULT_CARBON_DIALECT + "/displayName";